builder/config.go

```diff
@@ -47,3 +45,23 @@ func NewConfig(options *compileopts.Options) (*compileopts.Config, error) {
 		TestConfig:     options.TestConfig,
 	}, nil
 }
+
+func getGorootVersion() (major, minor int, err error) {
+	goroot := goenv.Get("GOROOT")
+	if goroot == "" {
+		return 0, 0, errors.New("cannot locate $GOROOT, please set it manually")
+	}
+
+	return goenv.GetGorootVersion(goroot)
+}
+
+func getGoVersion() (major, minor int, err error) {
+	ver, err := exec.Command("go", "version").CombinedOutput()
```

Comment:
Please use the `go` binary from the detected GOROOT. Something like:

```suggestion
	ver, err := exec.Command(filepath.Join(goroot, "bin", "go"), "version").CombinedOutput()
```

Reply:
Good point, but no longer relevant.